### ISSUE TITLE
slideshow view broken

### ISSUE THREAD

**Issue by hahihohehe** [darktable-org/darktable#2280]
When I open slideshow view the first image seems to be processed. After that the picture is not shown. If I then click on the mouse button to switch to the next photo the next photo is shown in one fourth of the screen.

**To Reproduce**
Just open slideshow

**Expected behavior**
Normally the photo should be displayed on the whole monitor, not just a tiny version in the upper left corner.

**Screenshots**
![Bildschirmfoto von 2019-03-24 19-16-05](https://user-images.githubusercontent.com/22681951/54883943-4d7cc300-4e6b-11e9-967e-742c3192a9a5.png)

**Platform (please complete the following information):**
 - OS: Ubuntu 18.04
 - Version 2.6.1

**Additional context**
My monitor is hidpi and scaled by factor two.

**Comment by TurboGit** [comment]
I have fixed some issues in the slideshow.

Yet, the first image being small is certain due to HIDPI screen and since I can't reproduce (no such screen) I suppose it is not fixed.

**Comment by TurboGit** [comment]
What is your screen resolution in pixels?

Have you changed the screen_dpi_overwrite in darktablerc? What is the output for:

`$ grep dpi ~/.config/darktable/darktablerc`

**Comment by hahihohehe** [comment]
Thank you for the fast reply.

My screen resolution is 3840x2160 pixels but it does not matter. I tested it in a VM with FHD resolution and it worked all fine. Then I changed system-wide scaling to 2 and there was the same problem that all the photos where only shown on one fourth of the screen. It also works all fine if I disable scaling in GNOME on my 4k monitor.
I did not change any settings in darktable regarding appearance or dpi.
screen_dpi_overwrite is -1.0
